scripts/partition_manager/partition_manager.rst

```diff
@@ -328,6 +331,100 @@ It includes :file:`autoconf.h` (which is generated by Kconfig) and uses a Kconfi
 
    #endif /* CONFIG_SOC_NRF9160 */
 
+.. _pm_regions:
+
+Regions
+=======
+
+The Partition Manager places partitions in different *regions*.
+For example, you can use regions for internal flash memory and external flash memory.
+These regions are predefined as ``flash_primary`` and ``external_flash``, respectively.
```

Comment:
Is that correct?

Reply:
Yes, you can use these references in your `pm.yml`. The external flash will only work if you have set the corresponding kconfig option.

Reply:
But as all of this is explained better in the "EXternal flash" chapter, IMO this does not give any value, so I will remove this sentence.

Reply:
OK, but then we need to make it clearer that some regions are predefined. When I first read this, I assumed that nothing is defined by default. I'll make some suggestions.